companies/hagengrote.json

```
@@ -0,0 +1,20 @@
+{
+    "slug": "hagengrote",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Hagen Grote GmbH",
+    "address": "Grüner Dyk 36\n47803 Krefeld\nDeutschland",
+    "phone": "+49 2151 607090",
```

Same for the phone number…

```suggestion
    "phone": "+49 2203 8984444",
```